src/FixerConfiguration/SubsetOf.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerConfiguration;
+
+/**
+ * @internal
+ */
+final class SubsetOf
+{
+    private $values;
+
+    public function __construct(array $values)
+    {
+        $this->values = $values;
+    }
+
+    public function getValues()
```

Comment:
what is the value of this class with passing instance of it over passing regular array?

Reply:
It serves as a marker when generating README content or describe command output. I first thought about using an interface but this class would have been the only one to implement it, which would have been a bit cumbersome IMO.